File: CS/DXExample.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;

namespace DXExample.Module.Win
{
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class DXExampleWindowsFormsModule : ModuleBase
    {
        public DXExampleWindowsFormsModule()
        {
            InitializeComponent();
        }
    }
}

File: VB/DXExample.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp

Namespace DXExample.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class DXExampleWindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
	End Class
End Namespace
